### DISCUSSION TITLE
How to do case sensitive search or case insensitive search manually?

### DISCUSSION THREAD

**Discussion by sgon00** [helix-editor/helix#5900] [Q&A] [answered]
How to do case sensitive search or case insensitive search manually?

By reading the documentation and issues, I don't really get how to achieve this.  Somebody talked about prefix search with `?i` and `?-i`. I don't really get how to use it. After pressing `/` to search, I tried use question mark symbol, but I failed to figure out how to do it.

It seems all lower case keyword will do case insensitive search and any capital letter in the keyword will make it case sensitive search. But I want a explicit control by myself. 

Thanks a lot.

**Comment by Proful** [comment, marked as answer]
Hope this helps!
![search_caseinsensitive](https://user-images.githubusercontent.com/354596/218065060-ddd95805-881f-4ad3-93b4-9490a6c4acf7.gif)

**Reply by sgon00** [reply]
Thanks a lot. Yeah, it does help. Basically, I just need to add brackets. :)

**Reply by goyalyashpal** [reply]
parenthesis* yes 😃

**Reply by SufianBabri** [reply]
Yes, wrapping in the parenthesis, e.g `(text to search)`

**Reply by adriangalilea** [reply]
Wish this was the default search tho.

Any way to config it? My brain is not likely to remember yet another one of those

**Reply by sigurdo** [reply]
Case sensitive search seems to be default if you configure `editor.search.smart-case` to `false`.
